docs/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## v0.3.0 - June 2022
```

Comment:
is this manually generated? should we use the releases page for this?

Reply:
This was something else that I was thinking about that snuck in here. Removing.

Reply:
looks like the file is still here?